install.sh

```diff
@@ -159,24 +181,23 @@ echo -en "${fg_cyan}Proceed? (y/N) $fg_reset"
 set +e
 read -r resp < /dev/tty
 set -e
-if [ "$resp" = y -o "$resp" = Y ]; then
+if [[ "$resp" =~ [Yy] ]]; then
   # find the session path for this session, write to session.env and start the service
-  uid=$(id -u)
-  session_id=$(dbus-send --system --print-reply --type=method_call \
+  session_id="$(dbus-send --system --print-reply --type=method_call \
     --dest=org.freedesktop.login1 /org/freedesktop/login1/session/auto \
     org.freedesktop.DBus.Properties.Get string:org.freedesktop.login1.Session string:Id | \
-    sed -n 's/.*string "\([0-9]\+\).*/\1/p')
-  if [ -n "$session_id" ]; then
-    session_path=$(dbus-send --system --print-reply --type=method_call \
+    sed -n 's/.*string "\([0-9]\+\).*/\1/p')"
```

Comment:
The double quotes around `$(.)` are unnecessary. `var=$(.)` is a single command and no command splitting will be done regardless of the output on the RHS.